SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -509,6 +475,51 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	PackageMap->Init(this, &TimerManager);
 }
 
+void USpatialNetDriver::CreateAndInitializeLoadBalancingClasses()
+{
+	const USpatialGDKSettings* SpatialSettings = GetDefault<USpatialGDKSettings>();
+	if (SpatialSettings->bEnableUnrealLoadBalancer == false)
```

Comment:
minor nit - I'd move the if check to where CreateAndInitializeLoadBalancingClasses is called, and just check the flag here.